ObsCoreExtensionForRadioData.tex

```diff
@@ -492,63 +484,28 @@ \section{The ivoa.obscore\_radio table}
 
 A TAP service that has \verb|ivoa.obscore_radio| must also have a table
 compliant to any version 1 of ObsCore, i.e., a table
-\verb|ivoa.obscore| containing only the basic ObsCore attributes.
-the two tables must share exactly the obs\_publisher\_did 
-column such
+\verb|ivoa.obscore| containing only the basic ObsCore attributes. The two tables must share exactly the \emph{obs\_publisher\_did} column such
 that a \verb|NATURAL JOIN| will yield per-dataset rows of obscore and
 radio extension metadata.  
-%Which columns these are is site-specific.
 
 To ensure that all compliant services can execute the same queries,
-all columns in tables~\ref{tab:ExtensionAtt} and \ref{tab:ExtensionAtt_instrumental} must be present in such a
-table, although any may be NULL. At least a foreign key into \verb|ivoa.obscore| will typically
-make the extension table user-visible. Additional free columns (such as \emph{f\_min}, \emph{f\_max} ) may also
-be added.\footnote{can we make rules such that such additional columns
-will not interfere with later extensions?}.
+all columns described in following tables ~\ref{tab:ExtensionAtt} , ~\ref{tab:ExtensionAtt_interferometry} and \ref{tab:ExtensionAtt_instrumental} must be gathered  in the \verb|ivoa.obscore_radio|
+table, although some of them may be NULL if no value apply . At least a foreign key into \verb|ivoa.obscore| will typically
+make the extension table user-visible. 
+%Additional free columns (such as \emph{f\_min}, \emph{f\_max} ) may also be added.
```

Comment:
Why suppressing this

Reply:
recent discussions about obscore : optional columns need extra processing on the client side . 
if we speak about f_min, f_max , they should be properly defined in the standard in order to be comparable . 
free  is anything while  optional is defined ans sometimes not filled.